README.md

```diff
@@ -191,6 +191,18 @@ You can specify a swagger nickname to use instead of the auto generated name by
 desc 'Get a full list of pets', nickname: 'getAllPets'
 ```
 
+## Overriding param type
+You can override paramType in POST|PUT methods to query, using the documentation hash.
+``` ruby
+params do 
+  requires :action, type: Symbol, values: [:PAUSE, :RESUME, :STOP], documentation: {param_type: 'query'}
```

Comment:
OCD: Put a space between `{` and `param_type` and `}`. My version of RuboCop for README :)